asebuque_chrystine/script.js

```diff
@@ -0,0 +1,40 @@
+document.addEventListener("DOMContentLoaded", function () {
+    const nameInput = document.getElementById("name");
```

Comment:
"apply also to other 'const' "
```suggestion
    let nameInput = document.getElementById("name");
```